lib/Renard/Curie/Model/Document/CairoImageSurface.pm

```diff
@@ -12,6 +12,14 @@ sub _build_last_page_number {
 	return scalar @{ $self->image_surfaces }
 }
 
+=method get_rendered_page
+
+Returns a new Renard::Curie::Model::Page::CairoImageSurface object.
+
+See L<Renard::Curie::Model::Document::Role::Renderable> for more details.
```

Comment:
Looks good!

Just be sure to add ```[ci skip]``` to all your commit messages (anywhere in the message) until the end of the PR so that it doesn't run the tests on Travis-CI and Appveyor each time.